### DISCUSSION TITLE
v5.0.0-beta2

### DISCUSSION THREAD

**Discussion by michaelliunsky** [michaelliunsky/niRvana-theme#13] [General]
**虽然是 v5.0 beta 版，但修复了重要 BUG，请务必更新！**
- 重写了主题 JS
- 修复了评论 BUG
- 优化提示

<hr /><em>This discussion was created from the release <a href='https://github.com/michaelliunsky/niRvana-theme/releases/tag/v5.0.0-beta2'>v5.0.0-beta2</a>.</em>

**Comment by wangdanding128** [comment]
安装完主题 有警告  Warning: count(): Parameter must be an array or an object that implements Countable in /www/wwwroot//wp-content/themes/niRvana/functions.php on line 968

**Reply by michaelliunsky** [reply]
确实有这个问题，目前方法如下：
1.如果**不是php7.4改用php7.4**
2.找到主题functions.php在**第一行后面**添加如下代码：
**error_reporting(0);**

**Reply by michaelliunsky** [reply]
- 目前我已修复bug，等下个beta版同步，最多一周内
**还是不怎么稳定，等到5.0正式版尽量兼容php8.X吧**
